File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        m = set(nums)
        res = 0
        for n in m:
            if (n-1) not in m:
                seq = 1
                while (n + seq) in m:
                    seq += 1
                res = max(res, seq)
        
        return res

            
            

File: Data Structures & Algorithms/replace-elements-with-greatest-element-on-right-side/submission-0.cpp
class Solution {
public:
    vector<int> replaceElements(vector<int>& arr) {
        for (int i = 0; i < arr.size(); i++){
            int m = 0;
            for (int j = i + 1; j < arr.size(); j++){
                m = max(m, arr[j]);
            }
            arr[i] = m;

        }
        int end = arr.size() - 1;
        arr[end] = -1;
        return arr;

        






    }
};